misc/plugutils.c

```diff
@@ -405,6 +407,104 @@ make_table_driven_lsc_info_json_str (const char *scan_id, const char *ip_str,
   return json_str;
 }
 
+/**
+ * @brief Get the nvti's severity vector and return the score as string.
+ *
+ * @param[in] nvti     The nvti where to get the severity vector from.
+ *
+ * @return string representing a float value, with max 1 decimal. Eg. -1, 3.2.
+ * NULL otherwise. Must be free()'d by the caller.
+ */
+static gchar *
+get_severity_score_from_vt (nvti_t *nvti)
+{
+  gchar *vector;
+  gchar *score_str;
+  double score;
+
+  vector = nvti_severity_vector_from_tag (nvti);
+  if (!vector)
+    return NULL;
+
+  score = get_cvss_score_from_base_metrics (vector);
+  g_free (vector);
+
+  score_str = g_strdup_printf ("%.1f", score);
+
+  return score_str;
+}
+
+/**
+ * @brief Get the nvti's QoD and return the value as
+ * string.
+ *
+ * @param[in] nvti The nvti where to get the QoD from.
+ *
+ * @return string representing an integer value, NULL on error.
+ *  Must be free()'d by the caller.
+ */
+static gchar *
+get_qod_from_vt (nvti_t *nvti)
+{
+  int qod;
+  gchar *qod_str, *qod_type;
+
+  qod_str = NULL;
+  qod_type = nvti_get_tag (nvti, "qod_type");
+  if (qod_type)
+    {
+      qod = qod_type2val (qod_type);
+      qod_str = g_strdup_printf ("%i", qod);
+      g_free (qod_type);
+      return qod_str;
+    }
+
+  qod_str = nvti_get_tag (nvti, "qod");
+  return qod_str;
+}
+
+/**
+ * @brief Build a json representation of the part related to nvti information
+ * of a result.
+ *
+ * OpenVAS result contains also the nvti name, QoD and score, which need to be
+ * taken from redis and the score must be calculated from the vector
+ *
+ * @param[in]  oid     The oid of the NVT.
+ * @param[out] builder Json builder to add data to.
+ *
+ */
+static void
+add_nvti_info_into_json_builder (const char *oid, JsonBuilder *builder)
```

Comment:
```suggestion
static JsonBuilder *
json_builder_add_nvti_info (JsonBuilder *builder, const char *oid)
```
Maybe something like that for consistency.